docset/winserver2012r2-ps/smbshare/Get-SmbShare.md

```diff
@@ -66,33 +66,32 @@ VMS4                          Contoso-SO                    C:\ClusterStorage\Vo
 VMS5                          *                             D:\VMS
 ```
 
-This example retrieves the SMB shares on the computer.
+This command retrieves the SMB shares on the computer.
 
-### EXAMPLE 2
+### Example 2: Get a specific SMB share on a the local computer
 ```
-PS C:\>Get-SmbShare -Name VMS1
+PS C:\>Get-SmbShare -Name "VMS1"
 Name                          ScopeName                     Path                          Description 
 ----                          ---------                     ----                          ----------- 
 VMS1                          Contoso-FS                    I:\VMS
 ```
 
-This example retrieves the SMB shares on the computer named VMS1.
+This command retrieves information about the SMB share named 'VMS1' on the local computer
```

Comment:
Add a period at the end of the sentence.

Reply:
Thank you. I have added a period at the end of the sentence.